typings/index.d.ts

```diff
@@ -2226,10 +2231,10 @@ declare module 'discord.js' {
 
   type Base64String = string;
 
-  type BitFieldResolvable<T extends string> =
-    | RecursiveReadonlyArray<T | number | Readonly<BitField<T>>>
+  type BitFieldResolvable<T extends string, N extends number | bigint> =
+    | RecursiveReadonlyArray<T | number | bigint | Readonly<BitField<T>>>
```

Comment:
```suggestion
    | RecursiveReadonlyArray<BitFieldResolvable<T, N>>>
```
i believe it should just be used like this

Reply:
the type would just be recursive at that point, TS doesn't like that in any case

Reply:
recursive types are allowed as of [TypeScript 3.7](https://typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#more-recursive-type-aliases)